src/bin/pg_autoctl/primary_standby.h

```diff
@@ -27,7 +27,7 @@ typedef struct LocalPostgresServer
 	PostgresSetup	postgresSetup;
 	bool			pgIsRunning;
 	char			pgsrSyncState[PGSR_SYNC_STATE_MAXLENGTH];
-	int64_t			walLag;
+	char            receivedLsn[NAMEDATALEN];
```

Comment:
Here too we will need a comment about why NAMEDATALEN is a good fit for the LSN value. Oh, and maybe some comments about why keeping the value as a string internally makes sense (I think it does, by the way).
So that would be a _section_ comment rather than in-line, I suppose.